src/Raven.Client/Json/BlittableJsonReader.cs

```diff
@@ -25,6 +26,18 @@ private class CurrentItem
         private readonly Action<JsonReader, State> _setState = ExpressionHelper.CreateFieldSetter<JsonReader, State>("_currentState");
         private readonly Action<JsonReader, JsonToken> _setToken = ExpressionHelper.CreateFieldSetter<JsonReader, JsonToken>("_tokenType");
 
+        public JsonOperationContext Context { get; }
```

Comment:
can change to readonly field
`public readonly JsonOperationContext Context`

Reply:
done